opmd_viewer/addons/pic/lpa_diagnostics.py

```diff
@@ -410,10 +410,25 @@ def get_laser_envelope( self, t=None, iteration=None, pol=None, m='all',
            Only used for thetaMode geometry
            The angle of the plane of observation, with respect to the x axis
 
-        slicing_dir : str, optional
-           Only used for 3dcartesian geometry
-           The direction along which to slice the data
-           Either 'x', 'y'
+        slicing : float or list of float, optional
+           Number(s) between -1 and 1 that indicate where to slice the data,
+           along the directions in `slicing_dir`
+           -1 : lower edge of the simulation box
```

Comment:
Just a comment: that's an interesting choice instead of a `[0.0 : 1.0]` range for the relative simulation box. Is there a specific reason for that?

Reply:
I kept it the same as before the PR. Both are fine for me.

Reply:
Ah I see, alright! Yes I personally prefer a length-one range with default 0.5 usually.
@RemiLehe since we break the API anyway, do we want to "simplify" the relative simulation box range on the way?

Reply:
Yeah, actually I think that I initially choose `-1` to `1` because the direction for slicing is oftentimes a transverse direction, where the bounds are symmetrical (e.g. `x` from -50 microns to +50 microns). In that case, I thought that it would be more intuitive for a user to have these bounds correspond to `-1` and `+1`. 

But it's true that this assumes a particular kind of simulation... We could change, but note that this extends the non-backward-compatibility, in the sense that 
```
get_field( 'E', 'x', slicing_dir='y', slicing=0)
```
is backward-compatible with the current PR, but would break with the proposed change.